spec/mailers/report_mailer_spec.rb

```diff
@@ -1,14 +1,21 @@
 # frozen_string_literal: true
 require 'rails_helper'
 
-RSpec.describe ReportMailer, type: :mailer do
+RSpec.describe ReportMailer, type: [:mailer, :aws_helpers] do
   describe '#year_to_date_report_email' do
-    subject { described_class.year_to_date_report_email('foo').deliver_now }
+    let(:filename) { 'foo' }
+    let(:mail) { described_class.year_to_date_report_email(filename).deliver_now }
+    subject do
```

Comment:
is the subject no longer used? You might look into a named subject if that's the case